odoo-shippable/scripts/build-image.sh

```diff
@@ -235,11 +235,22 @@ git_ps1_style(){
 PS1=$UserMachine$Color_Off$PathShort\$\\n"\$(git_ps1_style)"$Color_Off\$" "
 EOF
 
-# Add alias for multitail and configure
+# Add alias and function
 cat >> /etc/bash.bashrc << EOF
 alias tail2="multitail -cS odoo"
+alias rgrep="rgrep -n"
+git_fetch_pr() {
+    REMOTE=$1
```

Comment:
Could you receive a second parameter for the PR number?
If the PR number is not received then use `*` by default